src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ApplicantPrayer.java

```diff
@@ -53,6 +57,26 @@ public enum EndCivilPartnership implements HasLabel {
         private final String label;
     }
 
+    @Getter
+    @AllArgsConstructor
+    public enum JudicialSeparation implements HasLabel {
+
+        @JsonProperty("judicialSeparation") // Get correct text for the label
```

Comment:
Comment left over for correct text for labels. Are they done? If so can we remove the comments

Reply:
Done